msg/SensorProperties.msg

```diff
@@ -0,0 +1,7 @@
+string serial                  # Manufacturer specific serial number 
```

Comment:
```suggestion
string serial_no               # Manufacturer specific serial number 
```

Would be my personal preference, just to make sure that's not misinterpreted as something to do with serial links

Reply:
This would be fine by me.

Reply:
I've adapted this.